corehq/apps/linked_domain/tests/test_linked_apps.py

```diff
@@ -561,13 +561,16 @@ def test_overwrite_app_with_shadow_forms(self):
         linked_app = self._make_linked_app(upstream_app.domain)
 
         linked_app = overwrite_app(linked_app, upstream_app, {})
-        original_fingerprints = self._get_form_fingerprints(linked_app)
+        original_forms = sorted(linked_app.get_forms(), key=lambda form: form.name['en'])
```

Comment:
I was confused for a second because this code change doesn't seem to match the commit message.

But this is a good commit, makes the test easier to read.

Reply:
ah, I get it now